website/docs/components/application-state/partials/code/how-to-use.md

```diff
@@ -0,0 +1,91 @@
+## How to use this component
+
+This component currently supports two states: empty state and error state.
+
+### Empty state
+
+```handlebars
+<Hds::ApplicationState as |A|>
+  <A.Header @title="Empty state title text" />
+  <A.Body @text="The item you were looking for was not found." />
+</Hds::ApplicationState>
+```
+
+#### Empty state with a footer link
+
+```handlebars
+<Hds::ApplicationState as |A|>
+  <A.Header @title="Empty state title text" />
+  <A.Body @text="The item you were looking for was not found." />
+  <A.Footer as |F|>
+    <F.Link @icon="help" @text="Need Help" @href="/components/alert"
+    @iconPosition="trailing" />
+  </A.Footer>
+</Hds::ApplicationState>
+```
+
+#### Empty state with yielded body block
+
+```handlebars
+<Hds::ApplicationState as |A|>
+  <A.Header @title="Empty state title text" />
+  <A.Body>
+    <Doc::Placeholder @text="block yield" @height="100" @background="#eee" />
+  </A.Body>
+  <A.Footer as |F|>
+    <F.Link @icon="arrow-left" @text="Go back" @href="/" />
+  </A.Footer>
+</Hds::ApplicationState>
+```
+
+### Error state
+
+To indicate that the message is an error state, add `@subtitle` to the invocation.
```

Comment:
The "@subtitle" naming doesn't seem very intuitive to understand that it's meant to be used for errors. Would it possibly used for other types of non-error messages?

Reply:
I think the intention here was to be more flexible and allow for different things to be used other than error codes.

Reply:
Ok, I would instead explain that that for error states you should use the @subtitle to add an error code vs. saying that @subtitle indicates that it is an error which to me sounds confusing.

Reply:
After reviewing use cases and chatting with the designers, we've gone back to `@errorCode` 👍